blog/2016-12-05-easy-upgrades-processed-by-git.md

```diff
@@ -0,0 +1,125 @@
+---
+title: Easier Upgrades
+author: Nicolas Cuillery
+authorTitle: JavaScript consultant and trainer at Zenika
+authorURL: https://twitter.com/ncuillery
+authorImage: https://fr.gravatar.com/userimage/78328995/184460def705a160fd8edadc04f60eaf.jpg?size=128
+authorTwitter: ncuillery
+category: announcements
+---
+
+Upgrading to each new version of React Native has been painful. You might have seen something like this before:
+
+```
+$ react-native upgrade
+Upgrading project to react-native v0.38.0
+  conflict ios/MyAwesomeApp/AppDelegate.m
+? Overwrite ios/MyAwesomeApp/AppDelegate.m? (Ynaxdh)
+  y) overwrite
+  n) do not overwrite
+  a) overwrite this and all others
+  x) abort
+  d) show the differences between the old and the new
+  h) Help, list all options
+ ```
+
+None of the options is what we want. By overwriting the file we lose changes we've made locally. What if we tried to resolve the conflicts automatically instead?
+
+Today I am proud to introduce a new tool which makes upgrading easier. The tool is called `react-native-git-upgrade` and uses Git behind the scenes.
+
+## Usage
+
+Requirements: Git has to be available in the `PATH`. Your project doesn't have to be managed by Git.
```

Comment:
Suggestion: Quote character to wrap this in a orange block ?

Requirement should be singular, this is the only one 💪 

Preview (with "requirement:" bold):
<img width="627" alt="capture d ecran 2016-12-03 a 14 31 49" src="https://cloud.githubusercontent.com/assets/4425455/20859673/4c12778c-b965-11e6-8b1c-ce7076353bfa.png">